distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/SystemJvmOptions.java

```diff
@@ -86,42 +102,42 @@ static List<String> systemJvmOptions(Settings nodeSettings, final Map<String, St
      * that cgroup statistics are available for the container this process
      * will run in.
      */
-    private static String maybeOverrideDockerCgroup(String distroType) {
+    private static Stream<String> maybeOverrideDockerCgroup(String distroType) {
```

Comment:
This is a refactoring to make these more uniform. It's in [this commit](https://github.com/elastic/elasticsearch/pull/115527/commits/030cb0ac6c76a3e244ce0d6018932f5cef63d29c).